server/http/custom_routes.go

```diff
@@ -124,6 +127,42 @@ func (c *customController) GetTestVersionDefinitionFile(w http.ResponseWriter, r
 	w.Write(result.Body.([]byte))
 }
 
+// GetTransactionVersionDefinitionFile - Get the test definition as an YAML file
+func (c *customController) GetTransactionVersionDefinitionFile(w http.ResponseWriter, r *http.Request) {
```

Comment:
At some point I think it would be nice to have a better way to define custom routes without having to duplicate the most of the code everytime